File: This_App_Backend/src/main/java/com/example/This_App_Backend/dto/wishlistDTO/WishlistDto.java
package com.example.This_App_Backend.dto.wishlistDTO;

import lombok.*;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class WishlistDto {
    private Long wishlistId;
    private Long userId;
    private Long productId;
    private String productName;
    private String productImage;
    private Double productPrice;
    private Long storeId;
    private String storeName;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

}

File: this-app/src/app/models/wishlist-models/wishlistDTO.ts
export interface WishlistDto {
  wishlistId: number;
  userId: number;
  productId?: number;
  productName?: string;
  productImage?: string;
  productPrice?: number;
  storeId?: number;
  storeName?: string;
  storeImage?: string; // Add this if your API provides it
  createdAt: string;
  updatedAt: string;
}